dags/blocksec_plugin/ricochet_distribute_operator.py

```diff
@@ -1,19 +1,8 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
-import requests,json
-from time import sleep
+from blocksec_plugin.contract_interaction_operator import ContractInteractionOperator
+from blocksec_plugin.abis import DISTRIBUTE_ABI
```

Comment:
Create a `REX_ABI` and use the `StreamExchange` contract ABI, find it here: 0xeb367F6a0DDd531666D778BC096d212a235a6f78 ( this address has both `distribute` and `harvest`)